### PR TITLE
Remove Jackson imports from JacksonJsonUtils

### PR DESCRIPTION
**Pull request by artembilan**
Since the classes load their imports explicitly we can't use them for
classes which provides utilities to check classpath

* Move imports to fully qualified names in the target utility method

**Cherry-pick to 4.3.x**

### CODE REVIEW

File: spring-integration-core/src/main/java/org/springframework/integration/support/json/JacksonJsonUtils.java [hunk 1]
```diff
@@ -43,38 +36,39 @@ private JacksonJsonUtils() {
...
 	public static boolean isJackson2Present() {
-		return jackson2Present;
+		return JacksonPresent.isJackson2Present();
 	}
 
+	/**
+	 * @return true if Jackson 2 is present in the classpath
+	 * @deprecated since 4.3.10 in favor of {@link JacksonPresent}
+	 */
+
```
Comment (garyrussell) [inline spring-integration-core/src/main/java/org/springframework/integration/support/json/JacksonJsonUtils.java]: Needs `@Deprecated` - also checkstyle; will fix on merge.

File: spring-integration-core/src/main/java/org/springframework/integration/support/json/JacksonJsonUtils.java [hunk 2]
```diff
@@ -43,38 +36,39 @@ private JacksonJsonUtils() {
...
 	 */
-	public static ObjectMapper messagingAwareMapper() {
-		if (jackson2Present) {
-			ObjectMapper mapper = new ObjectMapper();
-			mapper.configure(MapperFeature.DEFAULT_VIEW_INCLUSION, false);
-			mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
-			mapper.enableDefaultTyping(ObjectMapper.DefaultTyping.NON_FINAL, JsonTypeInfo.As.PROPERTY);
+	public static com.fasterxml.jackson.databind.ObjectMapper messagingAwareMapper() {
+		if (JacksonPresent.isJackson2Present()) {
+			com.fasterxml.jackson.databind.ObjectMapper mapper = new com.fasterxml.jackson.databind.ObjectMapper();
```
Comment (garyrussell) [inline spring-integration-core/src/main/java/org/springframework/integration/support/json/JacksonJsonUtils.java]: No need to fully qualify now.

Comment (artembilan) [inline spring-integration-core/src/main/java/org/springframework/integration/support/json/JacksonJsonUtils.java]: Doh... Do you do that on merge as well?
Thanks then.

During cherry-pick there will be some conflicts with the `ConfigurableCompositeMessageConverter` - it's a new `5.0` class.

Comment (garyrussell) [inline spring-integration-core/src/main/java/org/springframework/integration/support/json/JacksonJsonUtils.java]: ok

**Comment by garyrussell** [comment]
Cherry-picked as 87e8beb, merged as 709cb7f on master after removing deprecated methods.

**Pull request closed without merging**